Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'erubis'
 gem 'devise_ldap_authenticatable' # needed to use LDAP instead of email auth
 
 gem 'pg'
+gem "passenger", ">= 5.0.25", require: "phusion_passenger/rack_handler"
```

Comment:
Because that's the production environment, and I wanted to make sure that I wasn't doing anything stupid in dev that wouldn't be noticed until production.